src/weather_magic/io.cljs

```diff
@@ -0,0 +1,58 @@
+(ns weather-magic.io
+  (:require-macros
+   [cljs.core.async.macros         :refer [go]])
+  (:require
+   [cljs-http.client               :as http]
+   [cljs.core.async                :refer [<!]]))
+
+(enable-console-print!)
+
+;; Requests example temperature data from thor webserver and prints it
```

Comment:
Den här kommentaren ska läggas in i funktionens dokumentationstext istället. Detsamma går för alla andra kommentarer i den här filen.